Source/Core/VideoCommon/RenderBase.h

```diff
@@ -171,6 +171,13 @@ class Renderer
   const MathUtil::Rectangle<int>& GetTargetRectangle() const { return m_target_rectangle; }
   float CalculateDrawAspectRatio() const;
 
+  // Crops the target rectangle to the framebuffer dimensions, reducing the size of the source
+  // rectangle if it is greater. Works even if the source and target rectangles don't have a
+  // 1:1 pixel mapping, scaling as appropriate.
+  void AdjustRectanglesToFitBounds(MathUtil::Rectangle<int>& target_rect,
```

Comment:
Our code-style says to pass pointers to objects that will be modified (for clarity at the call-site) rather than by-ref.
Athough it is pretty clear based on the name of this function.

Reply:
Ehh, consistency is best. Changed :)